docs/uptime-guide/overview.asciidoc

```diff
@@ -1,11 +1,56 @@
 [role="xpack"]
 [[uptime-overview]]
-== Overview
+== Elastic Uptime overview
 
-The Elastic Uptime solution built on the Elastic Stack.
-It allows you to monitor the availability and response time of software services and applications in real time.
+The Elastic Uptime solution is built on the Elastic Stack.
+Elastic Uptime allows you to monitor the availability and response times of applications and services in real time and to detect problems before they affect users.
+
+Elastic Uptime is based on {heartbeat-ref}/index.html[Heartbeat], an open source uptime monitor.
+Elastic Uptime is transparent, flexible, and developer-friendly.
```

Comment:
My feeling is that if they're reading the docs we don't need to sell them on it.

Reply:
Many people read the docs before they buy the product, so I don't have a problem in general if the introductory topic highlights some selling points. In this case, I inherited this sentence so I left it, but I agree it doesn't really add value here.

Reply:
Oh, I see what you mean, I didn't realize it was in the previous version. I'm fine keeping it for the purposes of this PR since changing it is out of scope. I think we should remove it in a subsequent one however.